### DISCUSSION TITLE
Make Sharp an opt-in dependency

### DISCUSSION THREAD

**Discussion by jmikrut** [payloadcms/payload#2467] [Roadmap]
In today's modern frontend ecosystem, image resizing generally happens with something like `next/image` - making it so Payload's forced use of Sharp can become completely unnecessary.

A user should still be able to opt-in to using Sharp, but we want to remove it from our dependencies by default and allow the user to install it if they desire its functionality.

**Comment by remy90** [comment]
https://nextjs.org/docs/messages/sharp-missing-in-production
